src/Node.php

```diff
@@ -72,7 +72,7 @@ public function setId(string $id): self
      */
     public function getName(): string
     {
-        if (empty($this->_name)) {
+        if (null === $this->_name) {
```

Comment:
If _name is '', what should happen?

Reply:
I'm pretty sure here the goal was to initialize the property as its default value is null. So when unitiliazed, the call to call getInfo is made, and when initiliazed it returns the previous value.